### PR TITLE
Add repos for Leap 42.2

### PR DESCRIPTION
**Pull request by okurz**
*Caution:* I don't have any idea what I am doing. Just bluntly applying

```
for i in $(ls *42.1*) ; do sed -e 's/42.1/42.2/g' -e 's/Leap_421/Leap_422/g'
    $i > ${i/42.1/42.2} ; done
```

which will maybe fix
https://openqa.opensuse.org/tests/225803#step/upgrade_select/9 and alike.

### CODE REVIEW

File: YaST/Repos/_openSUSE_Leap_42.2_Additional.xml.in [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+        <url>http://download.opensuse.org/repositories/X11:/lxde/openSUSE_Leap_42.2/</url>
+      </repository>
+      <repository recommended="false" format="rpm-md">
+        <_name>openSUSE BuildService - LibreOffice</_name>
+        <_summary>Latest stable LibreOffice release</_summary>
+        <_description>Provides you with the latest stable version of LibreOffice, the Office suite that openSUSE uses.</_description>
+        <!-- Check the version here each time for what is supposed to be set -->
+        <url>http://download.opensuse.org/repositories/LibreOffice:/5.0/openSUSE_42.2/</url>
+      </repository>
+LibreOffice repo not yet enabled for Leap 42.2 / dimstar / 20151006 -->
```
Comment (DimStar77) [inline YaST/Repos/_openSUSE_Leap_42.2_Additional.xml.in]: I refuse having said that in the context of 42.2 :)

Comment (okurz) [inline YaST/Repos/_openSUSE_Leap_42.2_Additional.xml.in]: Yes, I would assume. So does it still appply for Leap42.1 at all?

Comment (DimStar77) [inline YaST/Repos/_openSUSE_Leap_42.2_Additional.xml.in]: On Wed, 2016-07-20 at 14:47 -0700, Oliver Kurz wrote:
> > + http://download.opensuse.org/repositories/LibreOffice:/5.0/o
> penSUSE_42.2/;
> > +      </repository>
> > +LibreOffice repo not yet enabled for Leap 42.2 / dimstar /
> 20151006 -->
> Yes, I would assume. So does it still appply for Leap42.1 at all?
http://download.opensuse.org/repositories/LibreOffice:/5.1/openSUSE_Lea
p_42.1/
would exist.
> 
> 
> 
>   > 
>   > 

> 
> 

>

Comment (okurz) [inline YaST/Repos/_openSUSE_Leap_42.2_Additional.xml.in]: I don't know at all what this repos lists here are about at all. What do I need to do?

Comment (DimStar77) [inline YaST/Repos/_openSUSE_Leap_42.2_Additional.xml.in]: On Wed, 2016-07-20 at 22:40 -0700, Oliver Kurz wrote:
> 
> I don't know at all what this repos lists here are about at all. What
> do I need to do?
> 
> 
This is the list that shows when you go to YaST, Software Repositories,
Add, Community Repositories, Next
Let's focus on the Main Repos for now for Leap 42.2, keeping the others
disabled is 'good enough' for now

**Comment by okurz** [comment]
how can I know which repos to add and which not? I checked with

```bash
for i in $(sed -n 's/^.*url>\(.*\)<\/url>/\1/p' YaST/Repos/_openSUSE_Leap_42.2_Additional.xml YaST/Repos/_openSUSE_Leap_42.2_Additional.xml.in YaST/Repos/_openSUSE_Leap_42.2_Default.xml YaST/Repos/_openSUSE_Leap_42.2_Default.xml.in YaST/Repos/openSUSE_Leap_42.2_Servers.xml YaST/Repos/openSUSE_Leap_42.2_Servers.xml.in) ; do echo -n "$i: " && curl --silent $i | grep 'Error 404' ; done
```

which is telling me there are some repos but many also don't exist.

**Pull request closed without merging**